### ISSUE TITLE
1.3.1, 1.3.5 scribd pdf

### ISSUE THREAD

**Issue by chloecwh** [TWUOnline/psyc339#24]
> - [x] flagged above - 1.3.1 Activity - pdf not accessible [Treating Adult Survivors of Childhood Emotional Abuse and Neglect](https://twuonline.github.io/psyc339/assets/u1/Survivors_of_Emotional_Abuse_and_Neglect.pdf) (2021).  Replace with
> https://www.scribd.com/document/459294314/Treating-Adult-Survivors-of-Childhood-Emotional-Abuse-and-Neglect-pdf
> 
> Need some guidance from Production here.  
> - [ ] In 1.3.5 Activity: Case Study - there's a link to Chapter 2.  Can we create a copy of Ch.2 from the scribd online version?
> Perhaps we can flag copyright issues here.  I'd say accessibility trumps copyright.  thoughts? 
> 
> Unit 2
> - [x] 2.1.1 Activity - Add link to - Read In the Realm of Hungry Ghosts, pp. 1–3, and Chapters 11–16 (2010). [link to ebook](https://ebookcentral.proquest.com/lib/twu/detail.action?docID=6377146)
> 
> Unit 3
> - [x] 3.2.2 Activity - I'm trying to find a better copy than what we have: [Being Mortal: Chapter 1, The Independent Self](https://twuonline.github.io/psyc339/assets/u3//Being_Mortal_Chapter1.pdf) (2017).  I'll ask the library to scan this chapter.  In the meantime - perhaps [this ](https://archive.org/download/being-mortal_202212/Being%20Mortal.pdf) is a better version?  or [this](https://archive.org/download/beingmortalmedic0000atul/beingmortalmedic0000atul.pdf)?
> - [x] 3.3.3 Activity - says it's in git assets.  Could you check on this link, as it gives a 404 error msg  [Walking With God Through Pain and Suffering, Chapter 11](https://twuonline.github.io/psyc339/assets/u3/Timothy_Keller_Ch11.pdf) (2013)
>  

 _Originally posted by @kmarjanovic in [#9](https://github.com/TWUOnline/psyc339/issues/9#issuecomment-3344753380)_

I replaced both mention of Treating Adult Survivors of Childhood Emotional Abuse and Neglect with Scribd link as a temporary measure
